python/pdstools/reports/ModelReport.qmd

```diff
@@ -16,26 +16,35 @@ format:
     toc: true
     toc-title: Table of Contents
     theme:
-        light: flatly
-        dark: darkly
+      light: flatly
+      dark: darkly
 jupyter: python3
 ---
 
 ```{python}
+import sys
+sys.path.append('/Users/perdo/Documents/pega/pega-datascientist-tools/python')
```

Comment:
Don't forget to remove this at some point ;)

Reply:
:)

Reply:
Maybe you can build a test that fails if such lines are still there. I do something similar for the R version, a test fails if it finds un-commented library source inclusion.

Reply:
Right yeah I think something like that would be cool. Probably in tandem with adding default linters/formatters. I'll create an issue for that, wanted tot do it anyway